exporters/logging/src/test/java/io/opentelemetry/exporter/logging/LoggingSpanExporterTest.java

```diff
@@ -38,7 +39,7 @@
 import org.junit.jupiter.api.extension.RegisterExtension;
 import org.slf4j.event.Level;
 
-/** Tests for the {@link LoggingSpanExporter}. */
+@SuppressLogger(LoggingSpanExporter.class)
```

Comment:
I'm a bit confused on what is going on here. Whether this annotation is present or not I don't see log output for this when I run `./gradlew :exporters:logging:test`.

I thought it might be because the default log level is set to `warn`, but even after reducing it and removing the `@SurpressLogger` annotation, I don't see the logs produced by `LoggingSpanExporter`.

Reply:
Thanks for looking into this so deeply! I've never used a logging.properties file before and can't believe that I have to copy in the default values (e.g. ConsoleHandler) when using one... 🙀